NoSQL/README.md

```diff
@@ -5,50 +5,68 @@ Data is all around us and knowing how to use and manipulate databases is an incr
 ## The Goal 🥅
 **Who?** Built with the audience of college and high school students that have minimal to advanced coding experience in mind. Most do not have experience with NoSQL databases and deploying web applications.
 
-**What?** Concise and interactive Github Learning Lab that introduces and develops skills related to NoSQL databases with [MongoDB](https://azure.microsoft.com/en-us/services/functions/)
+**What?** Concise and interactive Github Learning Lab that introduces and develops skills related to NoSQL databases with [MongoDB](https://www.mongodb.com/)
+
+**How?** Students gain experience through building a complete web app with a frontend (HTML, CSS, JS, Atlas Realms) and backend (MongoDB, Azure Services, NodeJS). The application utilizes machine learning, an external API, and NoSQL databases to recommend products to users based on the contents in an uploaded image (objects and text within the picture). Authentication is used to incorporate the database, which stores a history of recommendations with a corresponding user.
 
-**How?** Students gain experience through building a complete web app with a frontend (HTML, CSS, JS) and backend (MongoDB, JS). [...]
 **When?** 
-* *Starting out:* ...
-* *The project:* ...
+* *Starting out:* Week 1 will ease students into the course by introducing them to MongoDB Compass and basic concepts related to NoSQL databases. They will also set up necessary tools (GitHub and Git Bash).
+* *The project:* Weeks 2 through 4 will focus on building the Product Recommendation Engine, walking through 3 basic steps: coding the backend Azure Function, deploying the frontend on Atlas Realms, and finally implementing the MongoDB database and authentication.
 
-**Where?** Tools used include Github, Git Bash, Atlas Realms, and MongoDB Databases.
+**Where?** Tools used include Github, Git Bash, Atlas Realms, Azure Services, and MongoDB (Compass) Databases.
 
 ### Project Description
-4-week course on using MongoDB, Atlas Realms, and Deploying a Webapp.
+4-week course on using MongoDB, Azure Services, Atlas Realms, and Deploying a Webapp.
```

Comment:
we are not using Atlas Realms

Reply:
@Land-dev should we have them deploy the web app somewhere or should we just use live server then?